src/core/eval.ml

```diff
@@ -138,6 +149,23 @@ and eq_modulo : term -> term -> bool = fun a b ->
     | (Appl(t1,u1), Appl(t2,u2)) -> eq_modulo ((u1,u2)::(t1,t2)::l)
     | (Meta(m1,a1), Meta(m2,a2)) when m1 == m2 ->
         eq_modulo (if a1 == a2 then l else List.add_array2 a1 a2 l)
+    (* Try a δ conversion *)
```

Comment:
I'd rather not use delta here (it is not very informative), just say `Try unfolding variable definitions.`.